### PR TITLE
Hotkeys Adjustmens

### PR DESCRIPTION
**Pull request by nkaradzhov**
- Change time units from ms to us for CPU time fields
- Add SlotRange interface for slot range representation
- Update field names (byCpuTime → byCpuTimeUs, etc.)
- Fix HOTKEYS_STOP to return null in empty state
- Update response parsing for new array-wrapped format
- Add test for HOTKEYS_STOP empty state behavior

---

### Checklist

- [ ] Does `npm test` pass with this change (including linting)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

### CODE REVIEW

File: packages/client/lib/commands/HOTKEYS_GET.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ function transformHotkeysGetReply(reply: UnwrapReply<HotkeysGetRawReply>): Hotke
       case 'selected-slots':
         result.selectedSlots = parseSlotRanges(value as unknown as Array<ArrayReply<NumberReply>>);
         break;
-      case 'sampled-command-selected-slots-us':
+      case 'sampled-commands-selected-slots-us':
         result.sampledCommandSelectedSlotsUs = Number(value);
```
Comment (PavelPashov) [inline packages/client/lib/commands/HOTKEYS_GET.ts]: Should we update the field in the response to `sampledCommandsSelectedSlotsUs` as well?

Comment (nkaradzhov) [reply packages/client/lib/commands/HOTKEYS_GET.ts]: totally, i thought i updated it...
